pkgsrc/switch-p4-16/api/common/pal.cpp

```diff
@@ -3526,14 +3526,14 @@ switch_status_t after_port_create(const switch_object_id_t object_id,
   port_lane_list.v_set(lane_list);
   status = switch_store::attribute_set(object_id, port_lane_list);
   if (status != SWITCH_STATUS_SUCCESS) {
-    switch_log(SWITCH_API_LEVEL_ERROR,
```

Comment:
Your commit comment for these was "Disable incorrect logging calls".  Why do you believe they are incorrect?

Does their presence in some log cause tests to fail, if they are not commented out?

If yes, is it because some code is checking for error messages in the logs, and failing the test if any are found?

I'm asking mainly because if we want to leave open the possibility of someone getting the necessary proprietary driver/BSP code from Intel and combine it with this code, these errors might not be incorrect at all in that scenario.

Reply:
I am temporarily disabling to see whether the switch memory compiles on Ubuntu 24.04. These calls do not work because they are incorrectly typed, which newer compilers seem to catch. This needs to be fixed.